<launch>
    <arg name="ground_plane" default="/ground_plane" />
    <arg name="queue_size" default="10" />
    <arg name="camera_namespace" default="/hardware/depth/kinect2" />
    <arg name="detected_persons" default="/yoloconvertor/detected_persons"/>
    <arg name="detection_id_offset" default="0"/>
    <arg name="detection_id_increment" default="1"/>
    <arg name="world_scale" default="1.0"/>
    <arg name="bounding_boxes" default="darknet_ros/bounding_boxes"/>
    <arg name="depth_scale" default="1.0"/>
    <arg name="pose_variance" default="0.01"/>

    <node pkg="yoloconvertor" type="yoloconvertor_standard" name="yoloconvertor_standard" output="screen">
        <param name="world_scale" value="$(arg world_scale)"/>
        <param name="ground_plane" value="$(arg ground_plane)" type="string"/>
        <param name="queue_size" value="$(arg queue_size)" type="int"/>
        <param name="camera_namespace" value="$(arg camera_namespace)" type="string"/>
        <param name="detected_persons" value="$(arg detected_persons)" type="string"/>
        <param name="detection_id_offset" value="$(arg detection_id_offset)"/>
        <param name="detection_id_increment" value="$(arg detection_id_increment)"/>
        <param name="bounding_boxes" value="$(arg bounding_boxes)"/>
        <param name="depth_scale" value="$(arg depth_scale)"/>
        <param name="pose_variance" value="$(arg pose_variance)"/>
    </node>

</launch> 
